### PR TITLE
Compose TitleBodyViews

### PR DESCRIPTION
**Pull request by emmaatkins**
# 📝 Description
Bold/H4/H5 TitleBodyViews

Github Issue
https://github.com/hmrc/android-components/issues/?
  
- [ ] Updated CHANGELOG
- [ ] Updated README

# 🛠 Testing Notes

# 📸 Screenshots

### CODE REVIEW

File: sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/data/repository/RepositoryImp.kt [hunk 1]
```diff
@@ -41,5 +46,8 @@ class RepositoryImpl @Inject constructor() : Repository {
 
         const val MOLECULE_INSET_VIEW = 1
         const val MOLECULE_INSET_TEXT_VIEW = 2
+        const val H4_TITLE_BODY_VIEW = 3
+        const val H5_TITLE_BODY_VIEW = 4
+        const val BOLD_TITLE_BODY_VUEW = 5
```
Comment (ngoulongkam) [inline sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/data/repository/RepositoryImp.kt:51]: Can we follow the naming pattern with "MOLECULE" at the front.

File: components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/titleBody/BoldTitleBodyView.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    headingText: String,
+    bodyText: String,
+    modifier: Modifier = Modifier,
+) {
+    Column {
+        Row(modifier = modifier) {
+            Text(
+                text = headingText,
+                style = HmrcTheme.typography.h6,
+                modifier = Modifier.padding(vertical = hmrc_spacing_8)
```
Comment (ngoulongkam) [inline components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/titleBody/BoldTitleBodyView.kt:38]: I think the spacing for this is incorrect.

I dont think we want a "vertical" spacing in this case as the component itself doesn't really have any padding at the top/bottom.

I guess we only want a padding 16 between this text and the one below it. 

(same comment applies to H4/5 component too)

File: sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/fragment/molecules/MoleculesFragment.kt [hunk 3]
```diff
@@ -61,6 +63,15 @@ class MoleculesFragment : Fragment(R.layout.fragment_molecules) {
                             MOLECULE_INSET_TEXT_VIEW -> {
                                 findNavController().navigate(R.id.action_moleculesFragment_to_insetTextViewFragment)
                             }
+                            BOLD_TITLE_BODY_VUEW -> {
```
Comment (ngoulongkam) [inline sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/fragment/molecules/MoleculesFragment.kt:66]: Spelling mistake "VIEW"

File: components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/titleBody/BoldTitleBodyView.kt [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                )
+        }
+        Row(modifier = modifier) {
+            Text(
+                text = bodyText,
+                style = HmrcTheme.typography.body,
```
Comment (sajinabraham) [inline components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/titleBody/BoldTitleBodyView.kt:44]:
```suggestion
                style = typography.body,
```
here too remove HmrcTheme

File: sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/BoldTitleBodyViewScreen.kt [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+                modifier = Modifier.padding(start = hmrc_spacing_16, end = hmrc_spacing_16)
+            )
+        }
+
+        Card(
+            modifier = Modifier
+                .padding(top = hmrc_spacing_16)
+                .fillMaxWidth(),
+            shape = RoundedCornerShape(0),
+            colors = CardDefaults.cardColors(containerColor = HmrcTheme.colors.hmrcWhiteBackground)
```
Comment (sajinabraham) [inline sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/BoldTitleBodyViewScreen.kt:74]: here also you can remove HmrcTheme

**Pull request merged**